org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/XtextUIMessages.java

```diff
@@ -39,7 +39,6 @@ private XtextUIMessages() {
 	}
 
 	/**
-	 * @param key
```

Comment:
so missing params are not complained?

Reply:
So far only *present* comments are validated. Enabling this for missing params would be ... quite some effort to be fixed.

I'd like to approach this step-by-step.
Unfortunately there is no such thing as validating only the touched files.